torchgen/model.py

```diff
@@ -806,6 +813,7 @@ def from_yaml(
                 has_composite_explicit_autograd_non_functional_kernel=has_composite_explicit_autograd_non_functional_kernel,
                 tags=tags,
                 namespace=namespace,
+                symint_args=symint_args,
```

Comment:
Validate that these argument names are actually in the schema in `__post_init__`